samples/bluetooth/central_hids/README.rst

```diff
@@ -1,22 +1,132 @@
+Basically, the sample simulates a computer that connects to a mouse or a keyboard.
 
 Overview
 ********
 
-Bluetooth central application that connects to the HID devices.
+The sample scans available devices, searching for a HIDS server.
+If any HIDS server is found, the sample connets to it and discovers all characteristics.
+
+If any input reports are detected, the sample subscribes to them to receive notifications.
+If any boot reports are detected, the behavior depends on if they are boot mouse reports or boot keyboard reports:
+
+* If a boot mouse report is detected, the sample subscribes to it.
+* If a boot keyboard report is detected, the sample subscribes to its input report, and it allows changing the CAPSLOCK LED on the keyboard (in boot mode).
+
 
 Requirements
 ************
 
-* BlueZ running on the host, or
-* A board with BLE support
+* One of the following development boards:
+
+  * nRF9160 DK board (PCA10090)
+  * nRF52840 Development Kit board (PCA10056)
+  * nRF52 Development Kit board (PCA10040)
+  * nRF51 Development Kit board (PCA10028)
+
+* HIDS device to connect with (for example, another board running the :ref:`peripheral_hids_mouse` or :ref:`peripheral_hids_keyboard` sample, or a Bluetooth Low Energy dongle and nRF Connect for Desktop)
+
+
+User interface
+**************
+
+Button 1:
+   Toggle the CAPSLOCK LED on the connected keyboard using Write without response.
+   This function is available only if the connected keyboard is set to work in Boot Protocol Mode.
+
+Button 2:
+   Switch between Boot Protocol Mode and Report Protocol Mode.
+   This function is available only if the connected peer supports the Protocol Mode Characteristic.
+
+Button 3:
+   Toggle the CAPSLOCK LED on the connected keyboard using Write with response.
+   This function is available only if the connected HID has boot keyboard reports.
+   It always writes CAPSLOCK information to the boot report, even if Report Protocol Mode is selected.
+
 
 Building and Running
 ********************
 
 This sample can be found under :file:`samples/bluetooth/central_hids` in the |NCS| folder structure.
 
 See :ref:`gs_programming` for information about how to build and program the application.
+
+
+Testing
+=======
+
+After programming the sample to your board, you can test it either by connecting to another board that is running the :ref:`peripheral_hids_keyboard` sample, or by using `nRF Connect for Desktop`_ that emulates a HIDS server.
+
+
+Testing with another board
+--------------------------
+
+1. Connect the board that runs this sample to a terminal (for example, PuTTY).
+   See :ref:`putty` for the required settings.
+#. Program the other board with the :ref:`peripheral_hids_keyboard` sample.
+#. Wait until the HIDS keyboard is detected by the central.
+   All detected descriptors are listed.
+   Check for information similar to the following::
+
+      HIDS is ready to work
+      Subscribe in report id: 1
```

Comment:
This should be "Subscribe to" - probably needs to be changed in the sample?

Reply:
Example and documentation would be updated.